Kickstarter-iOS/Tests/Views/LiveStreamContainerViewControllerTests.swift

```diff
@@ -23,11 +23,11 @@ internal final class LiveStreamContainerViewControllerTests: TestCase {
       |> Project.LiveStream.lens.startDate .~ (MockDate().timeIntervalSince1970 - 86_400)
       |> Project.LiveStream.lens.isLiveNow .~ false
     let liveStreamEvent = .template
-      |> LiveStreamEvent.lens.stream.hasReplay .~ true
```

Comment:
I got rid of the `Stream` sub-struct of `LiveStreamEvent` in order to flatten the models a bit.